.github/workflows/deploy.yml

```diff
@@ -57,12 +57,6 @@ jobs:
           make html BUILD=foreman-deb
           make html BUILD=satellite
 
-      - name: List files
-        run: find build
-
-      - name: Remove Last Modified
-        run: sed -i -E 's/^Last updated ([[:digit:]]+-[[:digit:]]+)-[[:digit:]]+ [[:digit:]]+:[[:digit:]]+:[[:digit:]]+ UTC$/Last updated \1/' build/*/*.html
-
```

Comment:
This hack is no longer needed since `Last updated` is now rendered correctly due to `-a docdatetime="$(DATE_MY)"` `Makefile` change.